packages/sol-compiler/src/solc_wrapper_v05.ts

```diff
@@ -0,0 +1,108 @@
+    compileDockerAsync,
+    compileSolcJSAsync,
+    getSolcJSAsync,
+    getSolidityVersionFromSolcVersion,
+    printCompilationErrorsAndWarnings,
+} from './utils/compiler';
+
+import { CompilationResult, ContractContentsByPath, ImportPrefixRemappings, SolcWrapper } from './solc_wrapper';
+
+// Solc compiler settings cannot be configured from the commandline.
+// If you need this configured, please create a `compiler.json` config file
+// with your desired configurations.
+export const DEFAULT_COMPILER_SETTINGS: solc.CompilerSettings = {
+    optimizer: {
+        enabled: false,
+    },
+    outputSelection: {
+        '*': {
+            '*': ['abi', 'evm.bytecode.object'],
+        },
+    },
+};
+
+// tslint:disable no-non-null-assertion
+
+export class SolcWrapperV05 extends SolcWrapper {
+    protected readonly _compilerSettings: solc.CompilerSettings;
+
+    public static normalizeOutput(
+        output: StandardOutput,
+        importRemappings: ImportPrefixRemappings,
+        opts: CompilerOptions,
+    ): StandardOutput {
+        const _output = _.cloneDeep(output);
+        // _output.sources = makeContractPathsRelative(_output.sources, opts.contractsDir!, importRemappings);
+        // _output.contracts = makeContractPathsRelative(_output.contracts, opts.contractsDir!, importRemappings);
+        // tslint:disable-next-line forin
+        for (const contractPath in _output.contracts) {
+            // tslint:disable-next-line forin
+            for (const contract of Object.values(_output.contracts[contractPath])) {
+                addHexPrefixToContractBytecode(contract);
+            }
+        }
+        return _output;
+    }
+
+    constructor(protected readonly _solcVersion: string, protected readonly _opts: CompilerOptions) {
+        super();
+        this._compilerSettings = {
+            ...DEFAULT_COMPILER_SETTINGS,
+            ..._opts.compilerSettings,
+        };
+    }
+
+    public get version(): string {
+        return this._solcVersion;
+    }
+
+    public get solidityVersion(): string {
+        return getSolidityVersionFromSolcVersion(this._solcVersion);
+    }
+
+    public areCompilerSettingsDifferent(settings: any): boolean {
+        return !_.isEqual(_.omit(settings, 'remappings'), _.omit(this._compilerSettings, 'remappings'));
+    }
+
+    public async compileAsync(
+        contractsByPath: ContractContentsByPath,
+        importRemappings: ImportPrefixRemappings,
+    ): Promise<CompilationResult> {
+        const input: solc.StandardInput = {
+            language: 'Solidity',
+            sources: {},
+            settings: {
+                remappings: [],
+                ...this._compilerSettings,
+            },
+        };
+        for (const [contractPath, contractContent] of Object.entries(contractsByPath)) {
+            input.sources[contractPath] = { content: contractContent };
+        }
+        for (const [prefix, _path] of Object.entries(importRemappings)) {
```

Comment:
Have we always supported import remappings? This is a pretty nice feature

Reply:
Yeah, this is nothing new.